src/tests/benchmark/docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '3'
+
+services:
+
+  crate:
+    image: crate:${CRATE_VERSION:-4.1.4}
+    command: crate -Cauth.host_based.enabled=false
+      -Ccluster.name=democluster -Chttp.cors.enabled=true -Chttp.cors.allow-origin="*"
+    ports:
+      # Admin UI
+      - "4200:4200"
+      # Transport protocol
+      - "4300:4300"
+    volumes:
+      - cratedata:/data
+
+  redis:
+    image: redis
+    ports:
+      - "6379:6379"
+
+  quantumleap:
+    image: ${QL_IMAGE:-smartsdk/quantumleap:latest}
+    command: --config server/gconfig_telemetry.py
+    sysctls:
+      net.core.somaxconn: 4096
```

Comment:
can this influence tests?

Reply:
Hopefully not, at least not that I can tell. The implementation is mostly lock-free and processes won't have to coordinate to write output files. Also, files only get written when a memory buffer fills up which normally would only happen when Gunicorn exits in our test cases. In fact, the memory buffer has a capacity of 1 MiB and collecting durations, GC & OS metrics for 10,000 request isn't enough to fill it up. 

I benchmarked request overhead with telemetry on. Like I said earlier, the average overhead for timing an HTTP request is less than one millisecond: 0.31 ms. You can also use the overhead gauge script in the tests directory to experiment yourself.

Reply:
my question was about this parameter `net.core.somaxconn: 4096`

Reply:
oh dear, that was a c&p from the notify benchmark :-)
I've found this: https://serverfault.com/questions/518862